fwutil/lib.py

```diff
@@ -357,6 +408,9 @@ def __parse_component_section(self, section, component, is_module_component=Fals
                 elif len(value1) == 2 and self.VERSION_KEY not in value1:
                     missing_key = self.VERSION_KEY
                     break
+                elif len(value1) == 3 and self.UTILITY_KEY not in value1:
+                    missing_key = self.UTILITY_KEY
+                    break
```

Comment:
As both `VERSION_KEY` and `UTILITY_KEY` are optional, please remove these checks since the above will break even if one of them is not present.